__tests__/models.test.js

```diff
@@ -135,6 +135,59 @@ describe("/api/reviews/:review_id/comments", () => {
   });
 });
 
+describe("POST: /api/reviews/:review_id/comments", () => {
+  it("should receive a response of 201 and return the posted comment ", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "mallionaire", body: "hello world" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toHaveProperty("comment_id", 7);
+        expect(body.comment).toHaveProperty("body", "hello world");
+        expect(body.comment).toHaveProperty("review_id", 1);
+        expect(body.comment).toHaveProperty("author", "mallionaire");
+        expect(body.comment).toHaveProperty("votes", 0);
+        expect(body.comment).toHaveProperty("created_at", expect.any(String));
+      });
+  });
+  it("should throw a 400 error when missing fields", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "mallionaire" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  it("should throw a 400 error when given a bad request", () => {
+    return request(app)
+      .post("/api/reviews/notAnId/comments")
+      .send({ username: "mallionaire", body: "hello world" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  it("should throw a 404 error when given an incorrect id", () => {
+    return request(app)
+      .post("/api/reviews/999999/comments")
+      .send({ username: "mallionaire", body: "hello world" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID can not be found");
+      });
+  });
+  it("should throw a 404 error when given an incorrect username", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "adam", body: "hello world" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID can not be found");
+      });
+  });
+});
+
```

Comment:
Really nice testing Adam, just a couple of extra considerations I've outlined in the comments above. I would also like you to consider one more test when posting: 
- Status 201, ignores unnecessary properties in the input body